x-pack/libbeat/common/otelbeat/otel.go

```diff
@@ -118,7 +119,7 @@ func getCollectorSettings(filename string) otelcol.CollectorSettings {
 	info := component.BuildInfo{
 		Command:     "otel",
 		Description: "Beats OTel",
-		Version:     "9.0.0",
+		Version:     "9.2.0",
```

Comment:
There is a package in libbeat that provides the version, is there any reason not to use it?
```suggestion
		Version:     version.GetDefaultVersion(),
```

The package is `libbeat/version`